src/main/java/com/elastic/support/diagnostics/DiagnosticInputs.java

```diff
@@ -89,7 +101,7 @@
 
     public final static String  typeDescription = "Enter the number of the diagnostic type to run.";
     public final static String  remoteUserDescription = "User account to be used for running system commands and obtaining logs. This account must have sufficient authority to run the commands and access the logs.";
-    public final static String  remotePasswordDescription = "Password for the remote login.";
+    public final static String  remotePasswordDescription = "Password for the remote login:";
```

Comment:
This come from an issue we had when testing the diagnostic with Ed. In this moment the diagnostic is requesting to the user to add the remote login, when we were running this, as this was a dot, we don't understand why the diagnostic stop there.
We though that the diagnostic get stuck and that was the last log displayed on the screen.
The colon will expect that will help the user to understand that we are asking for the password.

Reply:
BTW, if you run the diagnostic, before this question the OS will ask you the sudo password like this:
```
[sudo] password for camilo: 
```
So it's weird, the first one use colon and then we ask with a dot:
```
Password for the remote login.
```
For me it will make more sense if we follow the OS and when asking a question we use colon.